src/controllers/pricecheck.js

```diff
@@ -1,16 +1,15 @@
-'use strict';
+/* eslint-disable import/no-named-as-default, import/no-named-as-default-member */
```

Comment:
I don't like this anything we can do about it?

Reply:
i think that's a bad reason to get rid of it, but i've refactored it away and disabled the one problematic one at a project level.

Reply:
yeah having a bunch of these "dynamic rules" just seem like half baked solutions to me either you change the project structure or change the code to adhere to it, no point in linting if you follow it when it's only convenient (i'm being general, i'm not saying the pr is half baked or anything like that)